mpmt_afterpulse.cpp

```diff
@@ -153,46 +367,65 @@ int main( int argc, char* argv[] ) {
   std::cout<<"lsrCounter5: "<<lsrCounter5<<std::endl;
   std::cout<<"lsrCounter6: "<<lsrCounter6<<std::endl;
+  std::cout<<"lsrCounter7: "<<lsrCounter7<<std::endl;
+  std::cout<<"lsrCounter8: "<<lsrCounter8<<std::endl;
+  std::cout<<"lsrCounter9: "<<lsrCounter9<<std::endl;
+  std::cout<<"lsrCounter10: "<<lsrCounter10<<std::endl;
+  std::cout<<"lsrCounter11: "<<lsrCounter11<<std::endl;
+  std::cout<<"lsrCounter12: "<<lsrCounter12<<std::endl;
+  std::cout<<"lsrCounter13: "<<lsrCounter13<<std::endl;
+  std::cout<<"lsrCounter14: "<<lsrCounter14<<std::endl;
+  std::cout<<"lsrCounter15: "<<lsrCounter15<<std::endl;
+  std::cout<<"lsrCounter16: "<<lsrCounter16<<std::endl;
 
   std::cout<<"afpCounter1: "<<afpCounter1<<std::endl;
   std::cout<<"afpCounter2: "<<afpCounter2<<std::endl;
   std::cout<<"afpCounter3: "<<afpCounter3<<std::endl;
   std::cout<<"afpCounter4: "<<afpCounter4<<std::endl;
   std::cout<<"afpCounter5: "<<afpCounter5<<std::endl;
   std::cout<<"afpCounter6: "<<afpCounter6<<std::endl;
+  std::cout<<"afpCounter7: "<<afpCounter7<<std::endl;
+  std::cout<<"afpCounter8: "<<afpCounter8<<std::endl;
+  std::cout<<"afpCounter9: "<<afpCounter9<<std::endl;
+  std::cout<<"afpCounter10: "<<afpCounter10<<std::endl;
+  std::cout<<"afpCounter11: "<<afpCounter11<<std::endl;
+  std::cout<<"afpCounter12: "<<afpCounter12<<std::endl;
+  std::cout<<"afpCounter13: "<<afpCounter13<<std::endl;
+  std::cout<<"afpCounter14: "<<afpCounter14<<std::endl;
+  std::cout<<"afpCounter15: "<<afpCounter15<<std::endl;
+  std::cout<<"afpCounter16: "<<afpCounter16<<std::endl;
 
   // TIME histogram
   canvas1->cd();
   histT->SetTitle("Pulse times");
   histT->GetXaxis()->SetTitle("Time (ns)");
   histT->GetYaxis()->SetTitle("No. of events");
-  histT->GetYaxis()->SetRange(0,100);
+  histT->SetMaximum(1500);
   histT->Draw();
   canvas1->SaveAs(fnmT);
 
   // ALL PULSES histogram
   canvas2->cd();
-  histQ->SetTitle("Total pulses, charge");
-  histQ->GetXaxis()->SetTitle("Charge");
+  histQ->SetTitle("Total pulses");
+  histQ->GetXaxis()->SetTitle("Height (mV)");
   histQ->GetYaxis()->SetTitle("No. of events");
   histQ->GetXaxis()->SetRange(0,100);
   histQ->Draw();
   canvas2->SaveAs(fnmQ);
 
   // LASER PULSES histogram
   canvas3->cd();
-  histQlsr->SetTitle("Laser pulses, charge");
-  histQlsr->GetXaxis()->SetTitle("Charge");
+  histQlsr->SetTitle("Laser pulses");
+  histQlsr->GetXaxis()->SetTitle("Height (mV)");
   histQlsr->GetYaxis()->SetTitle("No. of events");
   histQlsr->GetXaxis()->SetRange(0,100);
   histQlsr->Draw();
   canvas3->SaveAs(fnmQlsr);  
-  canvas3->SaveAs("../Laser charges.png");  
 
   // AFTER PULSES histogram
   canvas4->cd();
-  histQafp->SetTitle("Afterpulses, charge");
-  histQafp->GetXaxis()->SetTitle("Charge");
+  histQafp->SetTitle("Afterpulses");
+  histQafp->GetXaxis()->SetTitle("Pulse (mV)");
   histQafp->GetYaxis()->SetTitle("No. of events");
   histQafp->GetXaxis()->SetRange(0,100);
   histQafp->Draw();
```

Comment:
Adjust threshold value and added the after pulse analysis script for a complete mPMT module (16?).